src/allocator/free_list_allocator/mod.rs

```diff
@@ -384,7 +384,10 @@ impl SubAllocator for FreeListAllocator {
                 chunk.offset,
                 chunk.allocation_type,
                 name,
-                chunk.backtrace
+                chunk
+                    .backtrace
+                    .as_ref()
+                    .map_or(&Backtrace::disabled(), |b| b)
```

Comment:
Aren't these equivalent to:
```suggestion
                chunk.backtrace.as_deref().unwrap_or(&Backtrace::disabled())
```

Reply:
implemented in both allocators